### ISSUE TITLE
Building: Stream.h included from ELF.h is missing

### ISSUE THREAD

**Issue by kode54** [jpd002/Play-#326]
Needs no comment, really. There is a file missing in the source. I am attempting to build the PsfPlayer application on Mac OS X.

E: Also missing are MemStream.h, StdStream.h, PtrStream.h, and StreamBitStream.h, and any associated .cpp files implementing those interfaces.

E2: Just noticed the existence of a Play--Framework repository that contains files needed for the build. My bad.

**Comment by jpd002** [comment]
I've updated the OSX PsfPlayer project to the "standard" other Xcode projects are using right now. So, basically, it should now work properly after cloning the Play-Build repo and building boost. 

Just so you know, the program kinda works, but I've never really completed that port, so, it's very primitive.

**Comment by kode54** [comment]
I was actually hoping to build something to use it as a PSF player component library, it just needs unique instances of the core component classes, and something to load PSX EXE or PSF2 filesystems into, and render sample data on demand.

Only problem is, I was hoping to be able to use it on an ARM 32 bit or 64 bit target as well, specifically iOS, and not jailbroken, so it'll need an interpreter core eventually.

Even if I can't have that yet, it'll be nice to have something that can play most or all PSF and PSF2 sets (and PSFP? I haven't seen any of those around yet.) without having to bundle BIOS code. It seems Audio Overload still has a ways to go.

**Comment by jpd002** [comment]
I've got everything needed to do ahead-of-time (AOT) compilation for platforms that don't support code generated at runtime, such as iOS, but it involves pre-processing all the PSF sets before hand to build an object file that will be linked in the player executable, so, it's not the best solution.

PSFP are PSP sound files. It's an unofficial extension the PSF format that I've created. I've only made one set though: http://purei.org/downloads/psfp/psportable.zip. I don't think many PSP games use synthesized music, so, it's probably pointless to have such a format.
